src/pathogen_embed/__init__.py

```diff
@@ -4,26 +4,22 @@
 Reduced dimension embeddings for pathogen sequences.
 """
 
-__version__ = "0.1.0"
+__version__ = "1.0.0"
 __author__ = 'Sravani Nanduri, John Huddleston'
 __credits__ = 'Bedford Lab, Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA'
 
 import argparse
 import sys
-from .embed import embed, get_hamming_distances
+from .pathogen_embed import pathogen_embed
 
 def make_parser():
     parser = argparse.ArgumentParser(description = "Reduced dimension embeddings for pathogen sequences", formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
+    parser.add_argument("--alignment", help="an aligned FASTA file to create a distance matrix with. Make sure the strain order in this file matches the order in the distance matrix.")
     parser.add_argument("--distance-matrix", help="a distance matrix that can be read in by pandas, index column as row 0")
     parser.add_argument("--separator", default=",", help="separator between columns in the given distance matrix")
-    parser.add_argument("--alignment", help="an aligned FASTA file to create a distance matrix with. Make sure the strain order in this file matches the order in the distance matrix.")
-    parser.add_argument("--cluster-data", help="The file (same separator as distance-matrix) that contains the distance threshold by which to cluster data in the embedding and assign labels given via HDBSCAN (https://hdbscan.readthedocs.io/en/latest/how_hdbscan_works.html). If no value is given in cluster-data or cluster-threshold, the default distance threshold of 0.0 will be used. If any additional columns in this file match embedding-specific parameters (e.g., 'n_components' or 'learning_rate'), the values from the first record of this file will override default values or values provided by the command line arguments.")
```

Comment:
We need to keep an argument like this `--cluster-data` argument to allow users (and our own workflows) to pass through a file with embedding method parameters that override the defaults. Since this command no longer performs clustering, we can rename the argument to something more accurate like `--embedding-parameters`.